packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -1,26 +1,87 @@
 import * as React from 'react';
 import { classNamesFunction, getId } from '@fluentui/react/lib/Utilities';
-import { ISankeyChartProps, ISankeyChartStyleProps, ISankeyChartStyles } from './SankeyChart.types';
+import { ISankeyChartData, ISankeyChartProps, ISankeyChartStyleProps, ISankeyChartStyles } from './SankeyChart.types';
 import { IProcessedStyleSet } from '@fluentui/react/lib/Styling';
 import * as d3Sankey from 'd3-sankey';
+import { area as d3Area, curveBumpX as d3CurveBasis } from 'd3-shape';
+import { sum as d3Sum } from 'd3-array';
+import { ChartHoverCard, IBasestate, SLink, SNode } from '../../index';
```

Comment:
The callout showing is taking like a second or so which is slower than other chart hover callouts. Can you check why this is happening and whether it can be improved.

Reply:
I have done the analysis for this and found out that createNodes() is taking the maximum time and inside createNodes, truncateText() was creating the slowness. This is fixed now Attaching the screenshots.
![Sankey_before changes](https://user-images.githubusercontent.com/103660888/212877627-9f04d138-8cad-4fda-85f6-b8caf3357b0e.png)
![sankeyAfterChanges](https://user-images.githubusercontent.com/103660888/212877666-ae3abc21-b69d-4a4e-91f2-91b0614bc2bb.png)